docs/locale/fr/LC_MESSAGES/userguide/troubleshooting.po

```diff
@@ -152,34 +157,34 @@ msgstr "Vous pouvez toujours supprimer ces appareils manuellement :"
 msgid "Go to Parsec configuration directory"
 msgstr "Aller dans le répertoire de configuration de Parsec"
 
-#: ../../userguide/troubleshooting.rst:46
+#: ../../userguide/troubleshooting.rst:47
 msgid ""
 "**On Windows**, open the file explorer and put ``%APPDATA%/parsec**`` in the "
```

Comment:
`parsec3`

Reply:
This was in the section "Specific to Parsec V2", but there is no point in keeping legacy doc. 
I'll update it.

Reply:
As discussed, we prefer to remove this section. We should instead add a function to properly remove devices.